common/i18n/resources/en.js

```diff
@@ -184,4 +192,38 @@ module.exports = {
       analysis: 'analysis',
     },
   },
+
+  categoryEdit: {
+    categoryName: 'Category name',
+    addLevel: 'Add level',
+    level: 'Level',
+    items: 'Items',
+    confirmDelete: 'Delete the item with all children? $t(common.cantUndoWarning)',
+    deleteItem: 'Delete item',
+  },
+
+  jobs: {
+    item: 'Item',
+    errors: 'Errors',
+
```

Comment:
should item and error plus plural go to common? what do you think?

Reply:
yes, better